imports/ui/cdp/CDPs.jsx

```diff
@@ -5,22 +5,49 @@ import List from '/imports/ui/cdp/List.jsx';
 import ChainStates from '../components/ChainStatesContainer.js'
 import { Helmet } from 'react-helmet';
 import i18n from 'meteor/universe:i18n';
+import {
+    TabContent, TabPane,
+    Nav, NavItem, NavLink,
+    
+} from 'reactstrap';
+import classnames from 'classnames';
 
 const T = i18n.createComponent();
 
-const CDPList = (props) => {
-    return <div>
-        <p className="lead"><T>cdp.cdpList</T></p>
-        <Row>
-            <Col md={12}>
-                <List {...props}/>
-            </Col>
-        </Row>
-    </div>
-}
+
 export default class CDPs extends Component{
     constructor(props){
         super(props);
+        this.state = {
+            cdpActiveTab: 'cdp-kava',
+            collateralType: 'ukava-a'
+        }
+    }
+
+
+    toggleTab = (tab, newCollateralType) => {
+        if (this.state.cdpActiveTab !== tab) {
+            this.setState({
```

Comment:
If the tabs are all the same, how about not switching tab here and only use one `TabPane` then use pass the correct prop to the `CDPList` component to get a new lis? I see you have `props` in `CDPList` but not passing any props into to it.

Reply:
Ok updated!